aleph-client/src/pallets/treasury.rs

```diff
@@ -91,25 +91,24 @@ impl TreasuryUserApi for SignedConnection {
 
 #[async_trait::async_trait]
 impl TreasurySudoApi for RootConnection {
-    async fn approve(&self, proposal_id: u32, status: TxStatus) -> anyhow::Result<BlockHash> {
+    async fn sudo_approve(&self, proposal_id: u32, status: TxStatus) -> anyhow::Result<BlockHash> {
```

Comment:
its kinda redundant to call it sudo as its already in the name of the trait imho

Reply:
right, but then `approve` becomes ambiguous if we import both traits

Reply:
You can still qualify method in such case

Reply:
I thought it will be more readable this way, but I can change it